ntbs-service/Pages/Notifications/NotificationsBase.cshtml.cs

```diff
@@ -16,24 +16,18 @@ public class NotifyError {
         public List<string> ErrorMessages { get; set; }
     }
 
-    public abstract class NotificationModelBase : ValidationModel
+    public abstract class NotificationEditModelBase : NotificationModelBase
     {
```

Comment:
Needed only by edit pages - will rename the file too but didn't want github to get confused.